packages/tools/webpack-fluid-loader/webpack.config.js

```diff
@@ -25,6 +25,8 @@ module.exports = {
       },
     ],
   },
+  // Webpack 5 does not support automatic polyfilling of node modules, setting node to false will help simulate webpack 5 behavior by throwing build errors when we rely on node polyfills
+  node: false,
```

Comment:
i wonder if this is causing the nconf issue, not sure if that is browser by default, or if it require node

Reply:
It doesn't repro on my laptop, where I never tried to do the sym link full think

Reply:
Though there is also a "process" implicit dependency, which needs to be sorted out too before merging this that would also block wp5